NOMENCLATURE.md

```diff
@@ -0,0 +1,76 @@
+# Cocoapods Nomenclature
+A list of terms used in the [cocoapods](https://github.com/CocoaPods/CocoaPods) context.
+
+## Glossary
+
+### Build
+A package generated after the code in your project has been compiled.
+
+### Bundle
+A bundle is a directory with a standardized hierarchical structure that holds executable code and the resources used by that code.
+
+### CocoaPods
+CocoaPods is a dependency manager for Swift and Objective-C Cocoa projects.
+
+### Dependency
+A file that is required by the project to execute a task.
+
+### Existing Xcode Project
+A subproject used directly to build a spec.
+
+### Framework
+A framework is a bundle that contains shared libraries as well as sub directories of headers and other resources.
+
+### Library
+Library is a packaged collection of executable object files that a program can link against.
+
+### Header
+The Header is a file that holds function declarations and macro definitions to be shared between several source files.
+
+### Native Target
+The `PBXNativeTarget` Xcode / Xcodeproj build artifact, which could be an application, a framework, or a static library, or any other target type supported by Xcode.
+
+### Pod
+An organized directory that manages the files to be used in a project along with its dependencies.
+
+### Podfile
+The Podfile is a specification that describes the dependencies of the targets of one or more Xcode projects.
+
+### Repo
+A virtual space where your code is stored.
+
+### Shared Library
+A Shared Library is a Library that is not copied into the project, instead, it's path is referenced by the project.
+
+### Static Library
+A Static Library is a Library that has been copied and bundled into the build.
```

Comment:
Replace "has been copied and bundled" with "statically linked" (and maybe link to something about static and dynamic linking?)